code/game/machinery/doors/airlock.dm

```diff
@@ -1217,16 +1221,36 @@ About the new airlock wires panel:
 			bashed_in(user, TRUE)
 		return
 	if(istype(I, /obj/item/tool/crowbar/halligan))
+		if (src.busy)
+			return
 		var/breaktime = 8 SECONDS
-		if(!operating && density && src.arePowerSystemsOn() && !((stat) & NOPOWER) && !welded)
-			if(locked && !lifted)
-				to_chat(user, "<span class='notice'>You begin to lift \the [src] out of its track, exposing the bolts.</span>")
-				playsound(src, 'sound/effects/rustle-metal.ogg', 50, 1)
-				if(do_after(user,src,breaktime))
-					to_chat(user, "<span class='notice'>You begin to lift the airlock out of its track, exposing the bolts.</span>")
-					playsound(src, 'sound/items/Deconstruct.ogg', 50, 1)
-					animate(src, pixel_y = pixel_y + 5, time = 1)
-					lifted = TRUE
+		if((!operating && src.arePowerSystemsOn() && !((stat) & NOPOWER) || locked) && !welded)
+			if(locked)
+				if (!lifted)
+					src.busy=1
+					to_chat(user, "<span class='notice'>You begin to lift \the [src] out of its track, exposing the bolts.</span>")
+					playsound(src, 'sound/effects/rustle-metal.ogg', 50, 1)
+					if(do_after(user,src,breaktime))
+						to_chat(user, "<span class='notice'>You lift the airlock out of its track, exposing the bolts.</span>")
+						playsound(src, 'sound/items/Deconstruct.ogg', 50, 1)
+						animate(src, pixel_y = pixel_y + 5, time = 1)
```

Comment:
`animate(src, pixel_y += 5, time = 1)`
my own shitcode...

Reply:
i'm not sure what you mean by this

Reply:
replace `pixel_y = pixel_y + 5` with `pixel_y += 5` for consistency